package.json

```diff
@@ -202,7 +203,8 @@
     "webpack-hot-middleware": "2.25.0",
     "webpack-merge": "5.7.3",
     "workspace-tools": "0.12.3",
-    "yargs": "13.3.2"
+    "yargs": "13.3.2",
+    "yargs-parser": "13.1.2"
```

Comment:
explicitly added dev dependency (used by `@nrwl/tao`), so lint rules will not yell. also we are using `yargs-parser` on other places directly -> it's a good practice to provide transient dependencies explicitly when they are being used directly